<launch>
  <node pkg="proto_ros" type="calib_stereo_node" name="calib_stereo_node" required="true" output="screen">
    <param name="config_file" value="$(find proto_ros)/config/calib_intel_d435i.yaml" />
    <param name="rosbag" value="/data/intel_d435i/imucam-1.bag" />
    <param name="cam0_topic" value="/stereo/camera0/image" />
    <param name="cam1_topic" value="/stereo/camera1/image" />
  </node>
</launch>
